packages/apps-config/src/api/spec/robonomics.ts

```diff
@@ -34,32 +31,6 @@ const definitions: OverrideBundleDefinition = {
           promisor_signature: 'MultiSignature'
         },
         LiabilityIndex: 'u32',
-        ValidationFunctionParams: {
```

Comment:
The (now) dangling comma on the previous line is what I'm assuming makes CI fail.

Reply:
Ah, yes, my bad. Fixed!